cordformation/src/main/kotlin/net/corda/plugins/DockerformUtils.kt

```diff
@@ -0,0 +1,41 @@
+package net.corda.plugins
+
+import org.yaml.snakeyaml.DumperOptions
+import org.yaml.snakeyaml.Yaml
+import org.yaml.snakeyaml.nodes.Tag
+import org.yaml.snakeyaml.representer.Represent
+import org.yaml.snakeyaml.representer.Representer
+import java.io.InputStream
+
+class DockerformUtils {
```

Comment:
If this approach is more reliable / **documented** than the `addImplicitResolver()` approach then fair enough. But in that case we don't need to create a `DockerformUtils` wrapper class.

Reply:
Removed DockerformUtils and put everything in back in Dockerform